include/libtorrent/file.hpp

```diff
@@ -187,6 +187,14 @@ namespace libtorrent
 // internal export should be used at unit tests only
 	TORRENT_EXTRA_EXPORT native_path_string convert_to_native_path_string(std::string const& path);
 
+// internal export should be used at unit tests only
+	TORRENT_EXTRA_EXPORT std::string convert_from_native_path(char const* s);
```

Comment:
i want to setup vanilla MINGW and new MinGW x86-x64 to check if this mingw cwd workaround can be dropped